projects/plugins/crm/src/automation/class-automation-exception.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * Adds the Automation_Exception class.
+ *
+ * @since $$next-version$$
  */
 class Automation_Exception extends \Exception {
 	const STEP_CLASS_NOT_FOUND    = 10;
```

Comment:
[The WP CS specifies](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs) that constants should also have a DocBlock (and that it includes a since tag as well):

```php
/**
 * Summary.
 *
 * @since x.x.x
 * @var type $var Description.
 */
const NAME = value;
```

The docs does say that the `@var` description may be omitted which I feel like we should do because it doesn't make a lot of sense 

```suggestion
	/**
	 * Step class not found exception code.
	 * 
	 * @since $$next-version$$
	 * @var int
	 */
	const STEP_CLASS_NOT_FOUND    = 10;
```